lms/static/sass/course/courseware/_sidebar.scss

```diff
@@ -1,234 +1,168 @@
 .course-index {
```

Comment:
@clrux, the diff in this file looks really weird. Can you add inline comments to specific adds/deletes that I should pay attention to as well as what you were doing there?